Formula/vim.rb

```diff
@@ -16,6 +16,7 @@ class Vim < Formula
   option "override-system-vi", "Override system vi"
   option "disable-nls", "Build vim without National Language Support (translated messages, keymaps)"
   option "with-client-server", "Enable client/server mode"
+  option "enable-termtruecolor", "Include support for 24-bit colors for compatible terminals"
```

Comment:
Is there any indication of which terminals are compatible, FWIW? What happens if I build with this option in an incompatible terminal?